packages/commons-test/src/testUtils.ts

```diff
@@ -302,3 +313,44 @@ export const getMockAuthData = (organizationId?: TenantId): AuthData => ({
   },
   selfcareId: generateId(),
 });
+
+export const getMockTokenStatesClientPurposeEntry = (
+  tokenStateEntryPK?: TokenGenerationStatesClientKidPurposePK
+): TokenGenerationStatesClientPurposeEntry => {
+  const clientId = generateId<ClientId>();
+  const purposeId = generateId<PurposeId>();
+  return {
+    PK:
+      tokenStateEntryPK ||
+      makeTokenGenerationStatesClientKidPurposePK({
+        clientId,
+        kid: `kid ${Math.random()}`,
+        purposeId,
+      }),
+    descriptorState: itemState.inactive,
+    descriptorAudience: "pagopa.it",
+    updatedAt: new Date().toISOString(),
+    consumerId: generateId(),
+    agreementId: generateId<AgreementId>(),
+    purposeVersionId: generateId<PurposeVersionId>(),
+    GSIPK_consumerId_eserviceId: makeGSIPKConsumerIdEServiceId({
+      consumerId: generateId<TenantId>(),
+      eserviceId: generateId<EServiceId>(),
```

Comment:
not sure if it is relevant to your tests, but these values should be the same used in the rest of the record

Reply:
Updated